app/views/templates/cover_letter_template.scala.html

```diff
@@ -496,40 +490,97 @@ <h2 class="heading-medium margin-top-small no-margin-bottom print-header__headin
             <div class="column-full">
                 <h3 class="heading-medium">@messages("pdf.ruling.section.ifYouDisagree.title")</h3>
                 <p class="govuk-body-s">
-                    If you disagree with this decision, you can either:
+                    @messages("pdf.ruling.section.ifYouDisagree")
                 </p>
-                <ul class="list list-bullet">
-                    <li>ask for a review with HMRC</li>
-                    <li>appeal to an independent tribunal</li>
+                <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                    <li>@messages("pdf.ruling.section.ifYouDisagree.1")</li>
+                    <li>@messages("pdf.ruling.section.ifYouDisagree.2")</li>
                 </ul>
                 <p class="govuk-body-s">
-                    You cannot ask for a review with HMRC and an appeal with an independent tribunal at the same time. However, you can appeal to an independent tribunal if you disagree with the outcome of your review.
+                    @messages("pdf.ruling.section.ifYouDisagree.within")
                 </p>
                 <p class="govuk-body-s">
-                    To find more information about reviews and appeals you can <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/877291/HMRC1.pdf">read the HMRC1 document</a> or phone 0300 200 3700.
+                    @messages("pdf.ruling.section.ifYouDisagree.cannot")
                 </p>
-                <h4 class="heading-small">Asking for a review with HMRC</h4>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.askForReview.title")</h4>
```

Comment:
`govuk-heading-s` makes the h4s a bit smaller than the paragraphs which is why I think the `heading-small` styling was created in this file. Please use the class `heading-small` instead, thanks